democluster/user-data

```diff
@@ -236,33 +57,36 @@ write_files:
 
     # Scheduling
     SchedulerType=sched/backfill
-    SelectType=select/cons_res
-    SelectTypeParameters=CR_Core
+    SelectType=select/cons_tres
+    SelectTypeParameters=CR_CPU_Memory
 
     # Logging
     SlurmctldDebug=3
     SlurmdDebug=3
-    DebugFlags=NO_CONF_HASH
-    JobCompType=jobcomp/none
 
     # Accounting
-    JobAcctGatherType=jobacct_gather/cgroup
-    JobAcctGatherFrequency="task=10"
-    AcctGatherNodeFreq=10
     AcctGatherProfileType=acct_gather_profile/influxdb
+    AcctGatherInterconnectType=acct_gather_interconnect/sysfs
```

Comment:
I just noticed this. It will break 100% the current job metrics implementation because both the jobbergate agent and API don't count on the following metrics:
```
InMB
OutMB
PacketsIn
PacketsOut
```

In general, I think the scope of the PR is to wide. We should have a dedicated feature task to include this change, so we guarantee that the other parts of Vantage will continue working.